### PR TITLE
Open-graph

### PR DESCRIPTION
**Pull request by davodesign84**

### CODE REVIEW

File: packages/kauri-web/components/containers/Article/ApprovedArticle/View.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ class ApprovedArticle extends React.Component<Props, State> {
   render () {
     const props = this.props
     if (!props.data.getArticle) return
-    const { title, id, content } = props.data.getArticle
+    const { title, id, content, attributes: { background } } = props.data.getArticle
```
Comment (rej156) [inline packages/kauri-web/components/containers/Article/ApprovedArticle/View.js]: Nullable

File: packages/kauri-web/components/containers/Article/ApprovedArticle/View.js [hunk 2]
```diff
@@ -78,10 +78,23 @@ class ApprovedArticle extends React.Component<Props, State> {
 
     return (
       <ArticleContent>
+        {console.log(background)}
```
Comment (rej156) [inline packages/kauri-web/components/containers/Article/ApprovedArticle/View.js]: console

File: packages/kauri-web/components/containers/Article/ApprovedArticle/View.js [hunk 3]
```diff
@@ -78,10 +78,23 @@ class ApprovedArticle extends React.Component<Props, State> {
...
         <Helmet>
           <title>{title} - Kauri</title>
           <meta name='keywords' content={articleKeywords.map(i => i)} />
           <link rel='canonical' href={`https://${hostName}/article/${id}/${slugify(title, { lower: true })}`} />
+          <meta property="og:title" content={title} />
```
Comment (rej156) [inline packages/kauri-web/components/containers/Article/ApprovedArticle/View.js]: content needs to be parsed methinks

File: packages/kauri-web/components/containers/Article/ApprovedArticle/View.js [hunk 4]
```diff
@@ -78,10 +78,23 @@ class ApprovedArticle extends React.Component<Props, State> {
...
+          <meta property="og:site_name" content="kauri.io" />
+          <meta property="og:url" content={`https://${hostName}/article/${id}/${slugify(title, { lower: true })}`} />
+          <meta property="og:description" content={`${content && content.substring(0,100)}...`} />
+          <meta property="og:type" content="article" />
+          <meta property="og:image" content={background || '/static/images/logo.svg'} />
+          <meta name="twitter:card" content="summary" />
+          <meta name="twitter:site" ccontent={`https://${hostName}/article/${id}/${slugify(title, { lower: true })}`} />
+          <meta name="twitter:title" content={title} />
+          <meta name="twitter:description" content={`${content && content.substring(0,100)}...`} />
+          <meta name="twitter:creator" content="@kauri_io" />
```
Comment (rej156) [inline packages/kauri-web/components/containers/Article/ApprovedArticle/View.js]: yass steal SEO

File: packages/kauri-web/components/containers/Article/ApprovedArticle/View.js (hunk 3 above)
Comment (davodesign84) [reply packages/kauri-web/components/containers/Article/ApprovedArticle/View.js]: Title should be fine :)

Comment (rej156) [reply packages/kauri-web/components/containers/Article/ApprovedArticle/View.js]: 👍

**Pull request merged**